railties/CHANGELOG.md

```diff
@@ -1 +1,8 @@
 Please check [5-1-stable](https://github.com/rails/rails/blob/5-1-stable/railties/CHANGELOG.md) for previous changes.
+
+*   Specify lang attribute in generated application layout
+
+    W3C recommends the use of the lang attribute on the html tag. This uses
+    the locale view helper to specify the current app's language.
+
+    *Edward Loveall*
```

Comment:
Thanks for the clarification, I was a little confused exactly how the CHANGELOG worked here. Fixed and repushed.